Artesian/_Query/MasQuery.py

```diff
@@ -1,56 +1,141 @@
+from Artesian import _ClientsExecutor
+from Artesian._ClientsExecutor import RequestExecutor
 from Artesian._Query.Query import _Query
 from Artesian._Query.QueryParameters.MasQueryParameters import MasQueryParameters
 from Artesian._Query.Config.ExtractionRangeConfig import ExtractionRangeConfig
 from Artesian._Query.Config.Granularity import Granularity
 from Artesian._Configuration.DefaultPartitionStrategy import DefaultPartitionStrategy
 import urllib
+from typing import List
+from __future__ import annotations
+
 class _MasQuery(_Query):
     __routePrefix = "mas"
-    def __init__(self, client, requestExecutor, partitionStrategy):
+    def __init__(self, client: _ClientsExecutor, requestExecutor: RequestExecutor, partitionStrategy: DefaultPartitionStrategy) -> None:
+        """ Inits _MasQuery 
+        
+        Args:
```

Comment:
these Args are not documented. either document or remove.
since this is a private ctor, documentation is not required